setup.py

```diff
@@ -25,7 +25,7 @@
     packages=["dask_jobqueue"],
     include_package_data=True,
     install_requires=install_requires,
-    tests_require=["pytest >= 2.7.1"],
```

Comment:
as tests_require  seems to be deprecated I changed this line to install the dependencies for the ci automatically. Should make it easier for normal devs as well

Reply:
Sounds good!